### PR TITLE
[FEATURE] Get started | Add Alert info in the Get started pop-up #129

### LINKED ISSUE #129: [FEATURE] Get started | Add Alert info in the Get started pop-up
**Is your feature request related to a problem?**
Alert info is missing in Get started popup

**What solution would you like?**
Add info about Alerts in the Get started popup
Step title: View security alerts
Description: 
After detectors are created, you can view alerts triggered by findings.
Buttons:  View alerts

**Do you have any additional context?**
Add alerts info as a third step, current Get started screenshot
get started

Get started with Alerts info added.
get started with alerts info

### PR DESCRIPTION
**Pull request by jovancacvetkovic**
### Description
Adds Alert info into Get started pop-up

### Issues Resolved
#129

### Screenshots
get started with alerts info

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx [hunk 1]
```diff
@@ -33,88 +24,86 @@ export const GettingStartedPopup: React.FC<GettingStartedPopupProps> = ({
...
-                  }}
-                >
-                  View findings
-                </EuiButton>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </div>
+          <GetStartedStep
+            title={
+              'After detectors are create, you can view insights and analyse security findings.'
```
Comment (amsiglan) [inline public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx:51]: Let's fix `create` to `created`

Comment (jovancacvetkovic) [reply public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx]: Changed to "created"

File: public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx [hunk 2]
```diff
@@ -33,88 +24,86 @@ export const GettingStartedPopup: React.FC<GettingStartedPopupProps> = ({
...
-                dismissPopup();
-                history.push(ROUTES.DETECTORS_CREATE);
-              }}
-            >
-              Create detector
-            </EuiButton>
-          </div>
+          <GetStartedStep
+            title={
+              'Identify security findings and threats from your log datas with curated detection rules.'
```
Comment (AWSHurneyt) [inline public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx:29]: This line seems to be missing some text. The mocks say "Identify security findings and threats from your log datas with detection rules. Additionally, you can set up alerts based on rule conditions."

File: public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx [hunk 3]
```diff
@@ -33,88 +24,86 @@ export const GettingStartedPopup: React.FC<GettingStartedPopupProps> = ({
...
+              },
+            ]}
+          />
+        ),
+      },
+      {
+        title: 'View security alerts',
+        children: (
+          <GetStartedStep
+            title={'After detectors are created, you can view alerts triggered by findings.'}
```
Comment (AWSHurneyt) [inline public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx:76]: This text doesn't quite match the mocks. The mock says "View alerts based on conditions you have specified from your detectors."

**Review by AWSHurneyt — COMMENTED** [review]
The mocks may have been updated a bit since this PR was initially created. Left some comments with the discrepancies I noticed.

File: public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx (hunk 3 above)
Comment (jovancacvetkovic) [reply public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx]: Updated based on https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/129#issuecomment-1322627758

**Review by AWSHurneyt — APPROVED** [review]

**Review by phestecAMZN — APPROVED** [review]

**Review by phestecAMZN — APPROVED** [review]

**Pull request merged**